docs/src/reference/index.md

```diff
@@ -1,5 +1,92 @@
 # API Reference
 
-Temporian's API Reference.
+This page gives an overview of all of Temporian's public symbols.
 
-<!-- TODO: write reference home page. Include table of operators. -->
+Check the index on the left for a more detailed description of any symbol.
+
+## Classes
+
+| Symbol                              | Description                                                                                    |
+| ----------------------------------- | ---------------------------------------------------------------------------------------------- |
+| [`tp.Node`][temporian.Node]         | Reference to the input or output of an operator in the compute graph.                          |
+| [`tp.EventSet`][temporian.EventSet] | Container for actual temporal data.                                                            |
+| [`tp.Schema`][temporian.Schema]     | Description of the data inside a [`Node`][temporian.Node] or [`EventSet`][temporian.EventSet]. |
+
+## Functions
+
+| Symbol                                    | Description                                                                            |
+| ----------------------------------------- | -------------------------------------------------------------------------------------- |
+| [`tp.evaluate()`][temporian.evaluate]     | Evaluates [`Nodes`][temporian.Node] on [`EventSets`][temporian.EventSet].              |
+| [`tp.plot()`][temporian.plot]             | Plots [`EventSets`][temporian.EventSet].                                               |
+| [`tp.event_set()`][temporian.event_set]   | Creates an [`EventSet`][temporian.EventSet] from arrays (list, numpy, pandas).         |
```

Comment:
nit: remove `pandas` from here (although it might work with pandas `Series`, I think it's more confusing than useful given `from_pandas`).